projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -862,7 +865,10 @@ export class IgxDatePickerComponent extends PickerBaseDirective implements Contr
 
         this._overlayService.onOpened.pipe(...this._overlaySubFilter).subscribe((eventArgs) => {
             this.opened.emit(eventArgs as IBaseEventArgs);
-            this._calendar?.daysView?.focusActiveDate();
+            const errors = DateTimeUtil.validateMinMax(this.value, this.minValue, this.maxValue);
+            if (Object.keys(errors).length === 0) {
+                this._calendar?.daysView?.focusActiveDate();
```

Comment:
Except, if you don't have an active date and you don't focus... the document active element goes to.. body? And then keyboard nav won't work?
Might make a lot of sense to check when setting the `viewDate` if the current value is out of the valid range, to perhaps shift the calendar to a valid date and show that instead

Reply:
That's the better option and the behavior will be consistent with the time picker - if you are greater than the max value, then after opening the max value in the calendar will be focused. Same thing for min value.